packages/core/src/lib/readCsv.ts

```diff
@@ -0,0 +1,34 @@
+import { CsvError, parse } from 'csv-parse/sync'
+
+import { Result } from './Result'
+
+type ParseCsvOptions = { delimiter?: string }
+export async function syncReadCsv(
+  file: File,
+  { delimiter = ';' }: ParseCsvOptions = {},
+) {
+  try {
+    const data = await file.text()
+    const records = parse(data, {
+      delimiter,
+      relax_column_count: true,
+      trim: true,
+      skip_empty_lines: true,
+      columns: true,
+      info: true,
+    }) as {
+      record: Record<string, string>
+      info: { columns: { name: string }[] }
+    }[] // not typed
```

Comment:
is this comment a TODO?

Reply:
Removing in other PR I just meant that this package doesn't type return